assets/docs/tulasiram58827/keras-ocr/dr/1.md

```diff
@@ -0,0 +1,24 @@
+# Lite tulasiram58827/keras-ocr/dr/1
+
+TF Lite quantized version of the Keras OCR Model for text recognition from images.
+
+<!-- parent-model: tulasiram58827/keras-ocr/1 -->
+<!-- asset-path: https://github.com/tulasiram58827/ocr_tflite/releases/download/v0.1/keras_ocr_dr.tar.xz -->
+
+[![Open Colab Notebook]](https://colab.research.google.com/github/tulasiram58827/ocr_tflite/blob/main/colabs/KERAS_OCR_TFLITE.ipynb)
+
+### About KerasOCR
```

Comment:
You also reflect the part on CTC Decoder here. And also consider including a sentence describing how it is slightly different. I think it might be better to introduce Keras OCR in one line and then bullet out the rest of the things.

Reply:
Updated.